src/mlpack/tests/test_function_tools.hpp

```diff
@@ -80,4 +81,45 @@ inline void LogisticRegressionTestData(arma::mat& data,
   }
 }
 
+inline void LoadBostonHousingDataset(arma::mat& trainData,
+                              arma::mat& testData,
+                              arma::Row<double>& trainLabels,
+                              arma::Row<double>& testLabels,
+                              data::DatasetInfo& info)
+{
+  arma::mat dataset;
+  arma::Row<double> labels;
+
+  if (!data::Load("boston_housing_price.csv", dataset, info))
+    FAIL("Cannot load test dataset boston_housing_price.csv!");
+  if (!data::Load("boston_housing_price_labels.csv", labels))
+    FAIL("Cannot load test dataset boston_housing_price_labels.csv!");
+
+  data::Split(dataset, labels, trainData, testData,
+      trainLabels, testLabels, 0.3);
+  // info.Type(3) = data::Datatype::categorical;
+  // info.Type(8) = data::Datatype::categorical;
+
+  // info.MapString<double>("0", 3);
+  // info.MapString<double>("1", 3);
+  // info.MapString<double>("1", 8);
+  // info.MapString<double>("2", 8);
+  // info.MapString<double>("3", 8);
+  // info.MapString<double>("4", 8);
+  // info.MapString<double>("5", 8);
+  // info.MapString<double>("6", 8);
+  // info.MapString<double>("7", 8);
+  // info.MapString<double>("8", 8);
+  // info.MapString<double>("24", 8);
+  // std::cout << arma::unique(trainData.row(8));
```

Comment:
If you choose to keep this, after #2980 is merged this should work fine.

Reply:
Yeah. Right now they throw an error, that's why they are commented. I will keep it 👍